test/e2e/cypress/e2e/hana_cluster_details.cy.js

```diff
@@ -193,6 +193,18 @@ context('HANA cluster details', () => {
             .eq(1)
             .should('have.class', host.status);
         });
+
+        it(`${host.hostname} icon should have the expected tooltip message`, () => {
```

Comment:
Hey @arbulu89 and @CDimonaco, I added a test, to check the tooltip message. 
We could extend this even and test different scenarios. For this, we would need to enchance the photofinish scenario and add some logic to validate it. 
What do you guys think ?